.github/workflows/performance.yml

```diff
@@ -0,0 +1,165 @@
+# This is a workflow triggered by PR or triggered manually
+# Runs quick performance tests and reports the comparison against HEAD
+# Test should take less than 10 minutes to run on current self-hosted devices
+name: "Performance Testing"
+
+# Controls when the action will run.
+# This workflow runs when manually triggered by keywords used in the start of a review comment
+# Currently that phrase is /bench_x64. /bench_aarch64 and /bench_both are TODOs.
+on:
+  pull_request_review:
+    types: [submitted, edited]
+  pull_request_review_comment:
+    types: [created, edited]
```

Comment:
The `Wasmtime_Repo_On_PR_Comment` job below filters on `github.event_name == 'pull_request_review'` so I think this action is never used as part of the trigger here?